### DISCUSSION TITLE
Having Preact instead of React makes use this framework really hard.

### DISCUSSION THREAD

**Discussion by arboledavargas** [freshframework/fresh#2785] [General]
Deno already learned the lesson, no body want to use your thing if it does not support the already existing libraries, and a lot of libraries don't work well in fresh, i think the advantages of using Preact does not worth the loose in DX.

**Comment by marvinhagemeister** [comment]
Fair point, sounds like you'd be happier with a different framework. Nothing wrong with that! There a lots of great frameworks apart from Fresh out there. Here at Deno we've heavily invested into getting npm support on par so that other frameworks work well in Fresh. It was one of the key tasks I helped with in the past months. So go out there, pick the framework you like most and launch your app/site 👍
